### ISSUE TITLE
Doesn't work with import aliases

### ISSUE THREAD

**Issue by edwinmarinn** [pjknkda/flake8-datetimez#12]
If I use an import alias the plugin doesn't detect the unsafe datetime usage

### Example
```python
import datetime as dt

print(dt.datetime.now()) # This should be flagged as an unsafe usage
```

**Comment by pjrulez** [comment]
@edwinmarinn does it apply to ruff's linter's [rules](https://docs.astral.sh/ruff/rules/#flake8-datetimez-dtz)?

Thanks for this project.
